data/json/items/vehicle/battery.json

```diff
@@ -0,0 +1,92 @@
+[
+  {
+    "id": "battery_car",
+    "type": "MAGAZINE",
+    "category": "veh_parts",
+    "name": "car battery",
+    "name_plural": "car batteries",
+    "description": "A 12v lead-acid battery used to power car electrical systems.",
+    "weight": 18000,
+    "volume": 4,
+    "price": 4000,
+    "material": [ "plastic" ],
+    "symbol": ":",
+    "color": "light_cyan",
+    "ammo_type": "battery",
+    "capacity": 4500,
+    "flags": [ "NO_SALVAGE", "NO_UNLOAD", "NO_RELOAD" ]
+  },
+  {
+    "id": "battery_motorbike",
+    "copy-from": "battery_car",
+    "type": "MAGAZINE",
+    "name": "motorbike battery",
+    "name_plural": "motorbike batteries",
+    "description": "A 12v lead-acid battery used to power smaller vehicles' electrical systems.",
+    "weight": 6000,
+    "volume": 3,
+    "price": 2000,
+    "capacity": 2000
+  },
+  {
+    "id": "battery_truck",
+    "copy-from": "battery_car",
+    "type":"MAGAZINE",
+    "category": "veh_parts",
+    "name": "truck battery",
+    "name_plural": "truck batteries",
+    "description": "A large 12v lead-acid battery used to power truck electrical systems.",
+    "weight": 30000,
+    "volume": 6,
+    "price": 6000,
+    "capacity": 8000
+  },
+  {
+    "id": "medium_storage_battery",
+    "copy-from": "storage_battery",
+    "type": "MAGAZINE",
+    "name": "medium storage battery",
+    "name_plural": "medium storage batteries",
+    "description": "A medium storage battery.",
+    "weight": 31000,
+    "volume": 10,
+    "price": 50000,
+    "price_postapoc": 8000,
+    "bashing": 14,
+    "capacity": 700
```

Comment:
@mugling hi, little typo here, medium storage battery used to have 7000 capacity